includes/admin/settings/register-settings.php

```diff
@@ -1364,26 +1364,17 @@ function edd_sanitize_html_class( $class = '' ) {
  */
 function edd_get_settings_tabs() {
```

Comment:
Wonder if we should add a `@since` to this docblock to note that as of this version, empty tabs are filtered out on display.